### PR TITLE
Radmin.VPN version 1.1.4393.14

### PR DESCRIPTION
**Pull request by felipejunio05**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22180)

### CODE REVIEW

File: manifests/r/Radmin/VPN/1.1.4393.14/Radmin.VPN.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Radmin.VPN
+PackageVersion: 1.1.4393.14
+InstallerSwitches:
+  Silent: /VERYSILENT /NORESTART
+  SilentWithProgress: /SILENT /NORESTART
+Installers:
+- Architecture: neutral
+  InstallerType: exe
```
Comment (OfficialEsco) [inline manifests/r/Radmin/VPN/1.1.4393.14/Radmin.VPN.installer.yaml:11]:
```suggestion
PackageVersion: 1.1.4393
InstallerSwitches:
  Custom: /NORESTART
Installers:
- Architecture: neutral
  InstallerType: inno
  Scope: machine
```

File: manifests/r/Radmin/VPN/1.1.4393.14/Radmin.VPN.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Radmin.VPN
+PackageVersion: 1.1.4393.14
+PackageLocale: en-US
+Publisher: Famatech Corp.
```
Comment (OfficialEsco) [inline manifests/r/Radmin/VPN/1.1.4393.14/Radmin.VPN.locale.en-US.yaml:7]:
```suggestion
PackageVersion: 1.1.4393
PackageLocale: en-US
Publisher: Famatech
```

File: manifests/r/Radmin/VPN/1.1.4393.14/Radmin.VPN.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Radmin.VPN
+PackageVersion: 1.1.4393.14
```
Comment (OfficialEsco) [inline manifests/r/Radmin/VPN/1.1.4393.14/Radmin.VPN.yaml:5]:
```suggestion
PackageVersion: 1.1.4393
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Remember to change the version folder name
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll need to update the version folder as well.

**Review by OfficialEsco — APPROVED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**